app/views/stores/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<hr>
+<h1>STORES</h1>
+<div style="display:grid; grid-template-columns: 1fr 1fr;justify-content: space-evenly; gap: 10px">
+  <% @stores.each do |store| %>
+    <div style="justify-self: right;">
+      <%= store.name %>
+    </div>
+    <div>
+    <%= button_to  "Detail", store, method: :get %>
+    </div>
+  <% end %>
+</div>
+<div>
+  <a href="<%= new_store_path %>">
+    <button>Create New Store</button>
+  </a>
```

Comment:
Just as a recommendation (you can dismiss this) try to use the button_to helper, including the GET method:
 ` <%= button_to 'Create New Store', new_store_path, method: :get %>`
```suggestion
  <%= button_to 'Create New Store', new_store_path, method: :get %>
```

Just a recommendation for future implementations